leak.c

```diff
@@ -0,0 +1,65 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+ 
+#define KB1     1024
+#define MEMSIZE (4 * KB1)
+ 
+char *string = "IDENTIFY 10 ISSUES IN THIS MEMLEAK PROGRAM";
+char *gstring = "IDENTIFY 10 ISSUES IN THIS MEMLEAK PROGRAM";
+char *gptr;
+char *rptr;
+int num;
+ 
+char * f1_alloc(void);
+char * f2_copyme(void);
+void   f3_free(char *, char *);
+ 
+main()
+{
+    srand((int)getpid());
+  //while (getpid() != 0) {
```

Comment:
Uncomment "while() body" and replace with while(1), getting memory leak in that case